lib/src/reducers/load_dna_file_reducer.dart

```diff
@@ -43,6 +43,8 @@ ${util.stack_trace_message_bug_report(stack_trace)}''';
         '\n* error type:    ${error.runtimeType}'
         '\n* error message: ${error.toString()}'
         '\n$hline'
+        '\nIf the file is imported from cadnano, use the opton of importing cadnano v2.'
```

Comment:
Be more specific than this. Imagine you are the user... they don't know the menu options, so tell them exactly how to import, e.g., "Select File --> Import cadnano v2".